core/block/editor/clipboard/clipboard.go

```diff
@@ -541,6 +543,21 @@ func (cb *clipboard) pasteFiles(ctx session.Context, req *pb.RpcBlockPasteReques
 	return blockIds, cb.Apply(s)
 }
 
+func getImageKind(filename string) model.ImageKind {
+	imageFormats := []string{"jpg", "jpeg", "png", "gif", "webp"}
```

Comment:
Why do you use these image formats? Looks like the magic values from somewhere

Reply:
There was a business logic error. We should not put autoAdded kind to images pasted via clipboard (not from html), so I removed this func :smiley: